packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx

```diff
@@ -266,12 +279,12 @@ const Template: React.FC<Props> = props => {
                   })
                 }}
               />
-            </Text>
+            </ItemSubtitle>
 
-            <Text color='grey600' size='12px' weight={500}>
+            <Text color='grey600' lineHeight='1.5' size='12px' weight={500}>
```

Comment:
need the `%` symbol or no?

Reply:
in this case no need, if you are about to say 150 then you need to set like `150%`